tests/test_md.py

```diff
@@ -314,17 +272,11 @@ def compute_reference_values():
 
         u_interval = 3
 
-        test_us, test_xs, test_boxes = ctxt_3.multiple_steps_U(lamb, num_steps, lambda_windows, u_interval, x_interval)
-        np.testing.assert_array_almost_equal(ref_all_lambda_us[u_interval::u_interval], test_us)
+        test_us, test_xs, test_boxes = ctxt_3.multiple_steps_U(num_steps, u_interval, x_interval)
+        np.testing.assert_array_almost_equal(ref_all_us[u_interval::u_interval], test_us)
 
         np.testing.assert_array_almost_equal(ref_all_xs[x_interval::x_interval], test_xs)
 
-        test_us, test_xs, test_boxes = ctxt_3.multiple_steps_U(
-            lamb, num_steps, np.array([], dtype=np.float64), u_interval, x_interval
-        )
```

Comment:
This is no longer testing a corner case with the removal of the `lambda_schedule` argument.